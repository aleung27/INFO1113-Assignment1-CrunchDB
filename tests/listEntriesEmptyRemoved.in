LIST ENTRIES
SET a 0
LIST ENTRIES
SET C 8
SET D 10
LIST ENTRIES
DEL C
LIST ENTRIES
PURGE a
PURGE D
LIST ENTRIES
