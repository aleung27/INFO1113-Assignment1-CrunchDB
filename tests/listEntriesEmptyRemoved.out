> no entries

> ok

> a [0]

> ok

> ok

> D [10]
C [8]
a [0]

> ok

> D [10]
a [0]

> ok

> ok

> no entries

